test/acceptance/test/utils_test.go

```diff
@@ -0,0 +1,12 @@
+package acceptance
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestFileExists(t *testing.T) {
+	require.True(t, FileExists("utils.go"))
```

Comment:
kind suggestion. It would be great to have a file used specifically for this test. In case the rare case utils.go changes name. There are some methods to create temp files like this:
```
tempFile, err := ioutil.TempFile("", "")
os.Remove(tempFile.Name())
```
Then we don't need to rely any external resources to test this function  🙂

Reply:
that is a very valid point